components/admin_console/billing/company_info_edit.tsx

```diff
@@ -28,16 +200,62 @@ const CompanyInfoEdit: React.FC<Props> = () => {
             </div>
             <div className='admin-console__wrapper'>
                 <div className='admin-console__content'>
-                    <div style={{border: '1px solid #000', width: '100%', height: '432px'}}>
-                        {'Edit Company Info View'}
+                    <div className='CompanyInfoEdit__card'>
+                        <div className='CompanyInfoEdit__form'>
+                            <div className='section-title'>
+                                <FormattedMessage
+                                    id='admin.billing.company_info_edit.companyDetails'
+                                    defaultMessage='Company Details'
+                                />
+                            </div>
+                            <div className='form-row'>
+                                <Input
+                                    name='companyName'
+                                    type='text'
+                                    value={companyName}
+                                    onChange={updateState(setCompanyName)}
+                                    placeholder={Utils.localizeMessage('admin.billing.company_info.companyName', 'Company name')}
+                                    required={true}
+                                />
+                            </div>
+                            <div className='form-row'>
+                                <Input
+                                    name='numEmployees'
+                                    type='text'
+                                    value={numEmployees}
+                                    onChange={updateState(setNumEmployees)}
+                                    placeholder={Utils.localizeMessage('admin.billing.company_info.numEmployees', 'Number of employees (optional)')}
+                                />
+                            </div>
+                            <div className='section-title'>
+                                <FormattedMessage
+                                    id='admin.billing.company_info_edit.company_address'
+                                    defaultMessage='Company Address'
+                                />
+                            </div>
+                            <div className='checkbox'>
+                                <label>
+                                    <input
+                                        type='checkbox'
+                                        checked={sameAsBillingAddress}
+                                        onChange={(event) => setSameAsBillingAddress(event.target.checked)}
+                                    />
+                                    <FormattedMessage
+                                        id='admin.billing.company_info_edit.sameAsBillingAddress'
+                                        defaultMessage='Same as Billing Address'
+                                    />
+                                </label>
```

Comment:
Should we show `Same as Billing Address` only if they have a billing address? If it's empty this option will not have anything to save 

<img width="1738" alt="Screen Shot 2020-10-16 at 12 36 13 PM" src="https://user-images.githubusercontent.com/936315/96290951-66047f00-0fb5-11eb-8fb5-6743de510127.png">